thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -113,7 +133,14 @@ type EvalFnRunner struct {
 	IncludeMetaResources bool
 	Ctx                  context.Context
 	Selector             kptfile.Selector
+	Exclusion            kptfile.Selector
 	dataItems            []string
+
+	// we will need to parse these values into Selector and Exclusion
+	selectorLabel []string
+	selectorAnno  []string
+	excludeLabel  []string
+	excludeAnno   []string
```

Comment:
nit: Can we use full name here `excludeAnnotations` ? I have found `Anno` to be very untuitive :)

Reply:
done